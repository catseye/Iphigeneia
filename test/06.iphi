(* Test 'while ... do ...' *)

var i in var j in begin
    i := 100;
    j := 0;
    while i > 0 do begin
        j := j + i;
	i := i - 1
    end
end
