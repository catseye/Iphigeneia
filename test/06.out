i=0
j=5050
